core/src/main/java/nu/yona/server/messaging/entities/Message.java

```diff
@@ -167,7 +169,8 @@ public void setRead(boolean isRead)
 	 * <ul>
 	 * <li>If isSentItem == true, it is logically the target user</li>
 	 * <li>The sender of this message</li>
-	 * <li>In case of a system-originated message (e.g. a goal conflict message), it is the user for which this message is sent</li>
+	 * <li>In case of a system-originated message (e.g. a goal conflict message), it is the user for which this message is
```

Comment:
Yes, line length. I don't know how this occurs, as we both use the same code style, but I occasionally see this.

Reverting it doesn't help, as it'll hapen again next time.